README.md

```diff
@@ -1,127 +1,116 @@
-Where to Find the Code
-======================
-To submit issues and patches please visit https://github.com/DataDog/chef-datadog.
-The code is licensed under the Apache License 2.0 (see  LICENSE for details).
+# Datadog Chef Cookbook
 
-[![Chef cookbook](https://img.shields.io/cookbook/v/datadog.svg?style=flat)](https://github.com/DataDog/chef-datadog)
-[![Build Status](https://img.shields.io/circleci/build/gh/DataDog/chef-datadog.svg)](https://circleci.com/gh/DataDog/chef-datadog)
-[![GitHub forks](https://img.shields.io/github/forks/DataDog/chef-datadog.svg)](https://github.com/DataDog/chef-datadog/network)
-[![GitHub stars](https://img.shields.io/github/stars/DataDog/chef-datadog.svg)](https://github.com/DataDog/chef-datadog/stargazers)
+The Datadog Chef recipes are used to deploy Datadog's components and configuration automatically. The cookbook includes support for:
 
-Datadog Cookbook
-================
+* Datadog Agent v7.x (default)
+* Datadog Agent v6.x
+* Datadog Agent v5.x
+* Log collection with Agent v6 & v7 (enable in [default.rb][1])
```

Comment:
```suggestion
```
I don't think we need to mention logs specifically (the same way we don't other Agent features like APM).